packages/specs/schema/src/components/open-spec/operationInQueryMapper.ts

```
@@ -33,37 +34,53 @@ function buildExamples(property: string, examples?: Record<string, OS3Example>)
   const schema = options.components?.schemas?.[name];
 
   if (schema && !options.oldSchemas?.[name]) {
-    delete options.components!.schemas![jsonParameter.$schema.getName()];
+    delete options.components?.schemas?.[jsonParameter.schema().itemSchema().getName()];
   }
 
   return Object.entries(schema?.properties || {}).reduce((params, [key, {description, ...prop}]: [string, any]) => {
+    const style = parameter.style || (prop.$ref && !getSchemaFromRef(prop.$ref, options)?.enum) ? "deepObject" : undefined;
+
     return [
       ...params,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)